tests/unit/core/models/test_extents.py

```diff
@@ -340,5 +341,18 @@ def test_type_checking(self):
         """
         input = "bad input"
 
-        with self.assertRaisesRegex(TypeError, "Expect polygon, received <class 'str'>"):
-            SpatialExtent(input)
\ No newline at end of file
+        with self.assertRaisesRegex(TypeError, "Expected polygon, received <class 'str'>"):
+            SpatialExtent(input)
+
+    def test_type_checking_crs(self):
+        """
+        GIVEN a non-CrsObject input
+        WHEN passed to SpatialExtent
+        THEN a TypeError is returned
+        Added due to https://metoffice.atlassian.net/browse/CAP-362
```

Comment:
I like the reference back to the ticket that prompted the work, very handy for understanding why we made this change